pkg/channel/testing/grpc_mock_client_conn.go

```diff
@@ -0,0 +1,22 @@
+package testing
```

Comment:
Pardon my ignorance here but does Go compiler ignore this automatically for the main binary?

Reply:
Yes, only imported packages are included in the binary.